v3/generate.go

```diff
@@ -0,0 +1,233 @@
+package v3
+
+import (
+	"io/ioutil"
+	"log"
+	"strings"
+
+	"github.com/Clever/yaml"
+)
+
+// Major version and minor versions are defined here.
+// May reuse this file for other/future API versions, but you may find it simpler
+// to copy this file and modify it for new API versions.
+const majorVersion = "3"
+
+var minorVersions = []string{"0"}
+var versionStrs = []string{}
```

Comment:
should this have `"v3.0"`?

Reply:
I add "v3.0" later, systematically, in `Generate()`, so we only need to have the minor versions listed once.

Reply:
oh I missed that line -- thanks!